modules/apps/src/main/java/com/axway/apim/appimport/ClientApplicationImportApp.java

```diff
@@ -111,5 +115,12 @@ public static void main(String args[]) {
 		int rc = importApp(args);
 		System.exit(rc);
 	}
+	private List<String> getCustomPropsFromDesiderdApp(ClientApplication desiredApp) {
+		if (desiredApp.getCustomProperties()!=null){
+			return (List<String>)desiredApp.getCustomProperties().keySet().stream().collect(Collectors.toList());
+			}
+		return null;
+	}
+
```

Comment:
I would prefer to see this code as part of the `JSONConfigClientAppAdapter` instead of the main application.

Reply:
sorry, I don't know if I understand well your requirement, I made a little refactoring and fixed the timeout error in test